client/js/_author/components/common/tiny_wrapper.jsx

```diff
@@ -89,7 +90,11 @@ export default class TinyWrapper extends React.Component {
           id={`${this.id}-tinymce`}
           content={this.props.text}
           config={this.tinyMCEConfig()}
-          onBlur={(e) => { this.props.onBlur(e.target.getContent(), e.target.isDirty()); }}
+          onBlur={(e) => {
+            const content = e.target.getContent();
+            const isChanged = e.target.isDirty() || _.isEmpty(content);
```

Comment:
Won't this trigger false updates? i.e they type something, then remove all of it and then blur, this will update even though the content hasn't really changed.